cita-executor/core/src/cita_executive.rs

```diff
@@ -154,11 +158,7 @@ impl<B: DB + 'static> CitaExecutive<B> {
                     data: t.data.clone(),
                 };
 
-                self.create(
-                    self.block_provider.clone(),
-                    self.state_provider.clone(),
-                    &params,
-                )
+                self.call_evm(&params)
```

Comment:
why  rename this? I think `create` may be better.

Reply:
I want to reuse the code with `call` function.